lib/node_modules/@stdlib/iter/every-by/README.md

```diff
@@ -167,7 +167,7 @@ console.log( bool );
 -   <span class="package-name">[`@stdlib/iter/every`][@stdlib/iter/every]</span><span class="delimiter">: </span><span class="description">test whether all iterated values are truthy.</span>
 -   <span class="package-name">[`@stdlib/iter/for-each`][@stdlib/iter/for-each]</span><span class="delimiter">: </span><span class="description">create an iterator which invokes a function for each iterated value before returning the iterated value.</span>
 -   <span class="package-name">[`@stdlib/iter/none-by`][@stdlib/iter/none-by]</span><span class="delimiter">: </span><span class="description">test whether every iterated value fails a test implemented by a predicate function.</span>
--   <span class="package-name">[`@stdlib/iter/some-by`][@stdlib/iter/some-by]</span><span class="delimiter">: </span><span class="description">test whether at least `n` iterated values pass a test implemented by a predicate function.</span>
+-   <span class="package-name">[`@stdlib/iter/some-by`][@stdlib/iter/some-by]</span><span class="delimiter">: </span><span class="description">test whether at least \`n\` iterated values pass a test implemented by a predicate function.</span>
```

Comment:
@Planeshifter I believe that when fixing the lack of escaping for `[` in descriptions, you ended up escaping all Markdown special characters. However, I don't know whether that should be the case.

Reply:
In this case, backticks are escaped, but likely do not need to be. And should a description include something like 

> ... some description `a*x + y` some more description...

it is not clear that the `*` should be escaped.

However, something like

> ...removes asterisk (*) characters.

should be escaped.

Reply:
@Planeshifter In short, I am not sure a naive approach can work here for escaping. Instead, one may need to use one of the `remark` family of tools to perform AST inspection.

Reply:
Similarly, something like

> ...supports fancy indexing `x[':,...,2:10']` and other things.

should also not be escaped.

Reply:
I don't understand the example with the asterisks nor the fancy indexing one. When we have an equation with multiplication signs, say `a*b + c*d`, surely we want to escape them as otherwise we end up with an italicized portion, `b+c` in this example? And adding a backslash after the `x` and before the `[` in the fancy indexing example doesn't hurt either.

I changed the escaping regex to not escape backticks for now:
https://github.com/stdlib-js/stdlib/commit/0ad7c780a7159176a09878ec4b6a4a45797f144b

Reply:
The issue I was pointing out in my examples is that, inside code blocks, characters should not be escaped. E.g., `a*b + c*d` vs `a\*b + c\*d`. See the issue?

A regex-based approach is inherently fragile when it comes to handling this situation.